stonesoup/models/transition/orbital/tests/test_orbit.py

```diff
@@ -0,0 +1,49 @@
+from ......types.groundtruth import GroundTruthPath
+from ......types.orbitalstate import TLEOrbitalState
+from ..orbit import SGP4TransitionModel
+
+# https://pypi.org/project/sgp4/
```

Comment:
This means that we need to go and install the sgp4 package I assume. There is a way of having this flagged as a dependency such that the flake8 tests work automatically.

Reply:
Yes the external sgp4 propagator library is required. Could please advise on how to add such a flag?

Reply:
@sdhiscocks, can you advise? I think required packages have to be added to a list somewhere.

Reply:
You can add dependencies to `setup.py`. Suggest you add a _orbital_ dependency set (you'll see examples for _dev_ and _video_ on _master_).

If you want to run them on CircleCI you'll also need to update `.circleci/config.yml` file. You'll see _dev_ there already, so can add more by delimiting them with comma e.g.:
```
pip install -e .[dev,orbital]
```

Reply:
In fact I modified .circleci/config.yml. Was that bad?

Reply:
It'd be better to change them as per my suggestion, as then it'll be easier if/when you add more dependencies, you just to it in _setup.py_ and CircleCI. Will also need astropy in setup.py, such when others pip install with _orbital_ extras, it'll pick the dependencies.